### PR TITLE
[ES] Add `People/Nomination Assessment Team/SEV rating`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/People/Nomination_Assessment_Team/SEV_rating/es.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+# Clasificación SEV
+
+El SEV es un sistema de puntuación usado por el [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (*NAT*) para determinar la relevancia del [restablecimiento de una nominación](/wiki/Beatmap_ranking_procedure#restablecimientos-de-nominaciones) para las evaluaciones de las nominaciones de los [Beatmap Nominators](/wiki/People/Beatmap_Nominators) (*BN*), proporcionando transparencia y consistencia. Este sistema de puntuación se divide en dos valores y se muestra como *Obviedad/Severidad*. Ambos valores varían entre 0 y 2, lo que lo convierte en un sistema fácil de entender y rápido de usar.
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/People/Nomination_Assessment_Team/SEV_rating/es.md:3]:
```suggestion
El SEV es un sistema de puntuación usado por el [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (*NAT*) para determinar la relevancia del [restablecimiento de una nominación](/wiki/Beatmap_ranking_procedure#restablecimientos-de-nominaciones) para las evaluaciones de las nominaciones de los [Beatmap Nominators](/wiki/People/Beatmap_Nominators) (*BN*), proporcionando transparencia y consistencia. Este sistema de puntuación se divide en dos valores y se muestra como *obviedad/severidad*. Ambos valores varían entre 0 y 2, lo que lo convierte en un sistema fácil de entender y rápido de usar.
```

File: wiki/People/Nomination_Assessment_Team/SEV_rating/es.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  - Añadir una línea roja para tener en cuenta el mod Nightcore
+  - Usar el doble o la mitad del BPM
+  - Compensación ligeramente incorrecta
+    - Para una temporización simple, menos de 6 ms
+    - Para una temporización compleja, menos de 10 ms
+- **1/0:** (11 %)
+  - Compás incorrecto
+- **1/1:** (49 %)
+  - Compensación incorrecta
+    - Para una temporización simple, alrededor de 6-12 ms
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/People/Nomination_Assessment_Team/SEV_rating/es.md:109]:
```suggestion
    - Para una temporización simple, alrededor de 6 a 12 ms
```

File: wiki/People/Nomination_Assessment_Team/SEV_rating/es.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+  - Objetos mal ajustados que apenas afectan a la jugabilidad
+    - Un slider end ligeramente desajustado que las herramientas podrían ayudar a encontrar
+    - Un objeto que se retrasa solo unos milisegundos
+- **1/1:** (42 %)
+  - Objetos mal ajustados que son difíciles de notar al jugar, pero que a veces causan juicios de 100
+- **1/2:** (8 %)
+  - Objetos mal ajustados que afectan notablemente a la jugabilidad
+    - Objetos mal ajustados que siempre causan un juicio de 100, a veces 50, o incluso el bloqueo de las notas
+    - Objetos mal ajustados que provocan un espaciado anormal entre las notas siguientes/anteriores
+    - Un objetos mal ajustado en una parte de un stream, burst o triple (que no se puede simplificar)
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/People/Nomination_Assessment_Team/SEV_rating/es.md:148]:
```suggestion
    - Un objeto mal ajustado en una parte de un stream, burst o triple (que no se puede simplificar)
```

**Review by Unnombredeusuarioextremadamentelargo — CHANGES_REQUESTED** [review]

File: wiki/People/Nomination_Assessment_Team/SEV_rating/es.md [hunk 4]
```diff
@@ -0,0 +1,161 @@
+# Puntuación SEV
```
Comment (agatemosu) [inline wiki/People/Nomination_Assessment_Team/SEV_rating/es.md:1]: Yo diría "calificación"

Comment (agatemosu) [inline wiki/People/Nomination_Assessment_Team/SEV_rating/es.md:1]: Yo diría "calificación"

Comment (agatemosu) [reply wiki/People/Nomination_Assessment_Team/SEV_rating/es.md:1]: o "nota"?

Comment (UnknownOsu) [reply wiki/People/Nomination_Assessment_Team/SEV_rating/es.md:1]: Me convence más `calificación`

Comment (agatemosu) [reply wiki/People/Nomination_Assessment_Team/SEV_rating/es.md:1]: pues esa

File: wiki/People/Nomination_Assessment_Team/SEV_rating/es.md [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+
+El SEV es un sistema de calificación usado por el [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (*NAT*) para determinar la relevancia del [restablecimiento de una nominación](/wiki/Beatmap_ranking_procedure#restablecimientos-de-nominaciones) para las evaluaciones de las nominaciones de los [Beatmap Nominators](/wiki/People/Beatmap_Nominators) (*BN*), proporcionando transparencia y consistencia. Este sistema de calificación se divide en dos valores y se muestra como *obviedad/severidad*. Ambos valores varían entre 0 y 2, lo que lo convierte en un sistema fácil de entender y rápido de usar.
+
+## Obviedad y severidad
+
+*Nota: Un restablecimiento hecho para arreglar cosas que se considerarían buenas si se dejaran sin arreglar siempre se puntúa con 0/0. Esto es para no desalentar el modding y la aplicación de mejoras en mapas [calificados](/wiki/Beatmap/Category#qualified).*
+
+La **obviedad** se refiere a la facilidad con la que se puede detectar el problema.
+
+| Valoración | Definición | Explicación |
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/People/Nomination_Assessment_Team/SEV_rating/es.md:11]:
```suggestion
| Calificación | Definición | Explicación |
```

File: wiki/People/Nomination_Assessment_Team/SEV_rating/es.md [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+| :-: | :-- | :-- |
+| 0 | No es obvio | Se aplica si el problema es poco claro o demasiado complejo como para detectarlo sistemáticamente. |
+| 1 | Se puede encontrar con experiencia | Requiere conocimiento/experiencia/atención para captarlo. Por lo general, las herramientas o los usuarios comunes no pueden encontrarlos, por ejemplo, problemas de temporización/metadatos. |
+| 2 | Se puede encontrar de un vistazo | Probablemente algo que podría detectar un usuario común, o que no se habría pasado por alto al comprobar y usar las herramientas correctamente. |
+
+La **severidad** se refiere a la repercusión del problema en el juego.
+
+| Valoración | Definición | Explicación |
```
Comment (Unnombredeusuarioextremadamentelargo) [inline wiki/People/Nomination_Assessment_Team/SEV_rating/es.md:19]:
```suggestion
| Calificación | Definición | Explicación |
```

**Review by Unnombredeusuarioextremadamentelargo — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**